units/modprobe@.service

```diff
@@ -8,8 +8,11 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Load kernel module %i
+Description=Load Kernel Module %i
```

Comment:
Maybe it's just me, but I like to keep the directives sorted alphabetically.

Reply:
So we do that for some unit but not for others. For most we put Description= first.

But it might make sense to look into this and order them all alphabetically, but if we do that we should probably do that across the board and probably have some test run as part of CI that refuses unordered unit files...

Anyway, I think for now it's ok to leave this unsorted, even though I sympathize with your point.